DOM+js+html/ui/script.js

```diff
@@ -0,0 +1,579 @@
+            "   <div class=\"hashtags\">\n" + allHashtags(photoPost.hashtags) + "</div>\n"
+            + photoPost.description +
+            "</div>";
+        let column = document.getElementById("column");
+        column.insertBefore(post, column.firstChild);
+        authorsSelect();
+        hashtagsSelect();
+        return 0;
+        // showPostsToHtml();
+    }
+
+    function deletePostFromHtml(id) {
+        let container = document.getElementById('column');
+        let post = document.getElementById(id);
+        if(!container.removeChild(post)){
+            return -1;
+        }
+        for(let i = 0; i < photoPosts.length; i++){
+            if(photoPosts[i].id == id){
+                photoPosts[i].isDeleted = true;
+                authorsSelect();
+                hashtagsSelect();
+                return 0;
+            }
+        }
+        // if (!js.removePhotoPost(id)) {
+        //     return -1;
+        // }
+        // showPostsToHtml();
+    }
+
+    function editPostFromHtml(id, change) {
+        if (!js.editPhotoPost(id, change)) {
+            return -1;
+        }
+        hashtagsSelect();
+        showPostsToHtml();
+    }
+
+
+    function authorsSelect() {
+        let set = new Set();
+        for (let i = 0; i < photoPosts.length; i++) {
+            if(photoPosts[i].isDeleted === false) {
+                set.add(photoPosts[i].author);
+            }
+        }
+        let an = document.getElementById("authornames");
+        while (an.firstChild) {
+            an.removeChild(an.firstChild);
+        }
+        set.forEach(function (value) {
+            an.innerHTML += "<option>" + value + "</option>";
+        });
+    }
+
+    function hashtagsSelect() {
+        let set = new Set();
+        for (let i = 0; i < photoPosts.length; i++) {
+            if(photoPosts[i].isDeleted === false) {
+                photoPosts[i].hashtags.forEach(function (value) {
+                    set.add(value);
+                });
+            }
+        }
+        let an = document.getElementById("hashtags");
+        while (an.firstChild) {
+            an.removeChild(an.firstChild);
+        }
+        set.forEach(function (value) {
+            an.innerHTML += "<option>" + value + "</option>";
+        });
+    }
+
+    function addLike(id, author) {
+        if (!js.editPhotoPost(id, {likes: new Set(author)})) {
+            return -1;
+        }
+        let post = document.getElementById(id);
+        if(post === null){
+            return -1;
+        }
+        let lc = post.getElementsByClassName("likesCount")[0];
+        for (let i = 0; i < photoPosts.length; i++) {
+            if (photoPosts[i].id == id) {
+                lc.innerHTML = photoPosts[i].likes.size.toString();
+                break;
+            }
+        }
+    }
+
+    function getTime(createdAt) {
+        let date = new Date(createdAt);
+        let days = date.getDate();
+        if (days < 10) {
+            days = '0' + days;
```

Comment:
I believe, a such functionality (date formatting) is already existed. Check object ``Date`` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date